### DISCUSSION TITLE
内存泄漏

### DISCUSSION THREAD

**Discussion by cmathx** [PaddlePaddle/PaddleOCR#13088] [Q&A]
#### 问题描述 / Problem Description
使用最新版本的paddleocr，内存依旧会泄漏，应该怎么解决

#### 运行环境 / Runtime Environment
- OS:debian linux
- Paddle:
- PaddleOCR:2.8.0

**Comment by SWHL** [comment]
这个应该是Paddle框架的问题，和PaddleOCR问题不大。
如果只推理的话，可以用[`rapidocr_onnxruntime`](https://github.com/RapidAI/RapidOCR)，这个不泄露
